portfolio/src/main/webapp/script.js

```diff
@@ -155,3 +155,30 @@ function updateMovieText(data) {
 
   return unorderedList;
 }
+
+function displayName(){
+  fetch('/data')
+  .then((response) => response.text()
+    
```

Comment:
Remove the commented code and console.log statements

Reply:
Done.